packages/ui-rating/src/Rating/index.tsx

```diff
@@ -77,38 +78,27 @@ class Rating extends Component<RatingProps> {
     const { valueNow, iconCount, valueMax } = this.props
 
     // prevent divide by zero errors
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    const max = valueMax > 0 ? valueMax : iconCount
+    const max = !!valueMax && valueMax > 0 ? valueMax : iconCount
 
-    // @ts-expect-error ts-migrate(2532) FIXME: Object is possibly 'undefined'.
-    const filledIcons = Math.round((valueNow * iconCount) / max)
+    const filledIcons = Math.round((valueNow! * iconCount!) / max!)
```

Comment:
If you fix the type of `max`, the `!` is not needed here

```suggestion
    const filledIcons = Math.round((valueNow! * iconCount!) / max)
```